src/components/relationships-panel/RelationshipsPanel.vue

```diff
@@ -6,41 +6,56 @@
           <v-checkbox id="heir-legal-rep-checkbox" class="mt-0"
             v-model="selectedRelationships"
             :value="RelationshipTypes.HEIR_LEGAL_REP"
-            :label="RelationshipTypes.HEIR_LEGAL_REP"
             :error="displayErrorState"
-          />
+          >
+            <template v-slot:label>
+              <span class="vuetify-label">{{ RelationshipTypes.HEIR_LEGAL_REP }}</span>
+            </template>
+          </v-checkbox>
```

Comment:
Use the label slot to change theme of the label.

Reply:
Nice.

Reply:
Thanks @severinbeauvais I did consider using `:deep()` but I thought this was simpler and easier to understand.  I'd be very happy to consider an alternative approach if using the slot makes you at all uncomfortable.

Reply:
This is easier to understand. Leave it.